CSA.md

```diff
@@ -1,215 +1,221 @@
 
 1. ## Service  
 
-    1.1 <span class="header_3">Access and Use.</span>  During the <span class="orderform_link">Subscription Period</span> and subject to the <span class="orderform_link">Use Limitations</span>, <span class="coverpage_link">Customer</span> may access and use the Cloud Service for its internal business purposes. If the Cloud Service includes Software or Documentation, <span class="coverpage_link">Customer</span> may copy and use the Software and Documentation only as needed to access and use the Cloud Service and only if <span class="coverpage_link">Customer</span> complies with the terms of this Agreement.
+    1.1 <span class="header_3">Access and Use.</span>  During the <span class="orderform_link">Subscription Period</span> and subject to the <span class="orderform_link">Use Limitations</span>, <span class="keyterms_link">Customer</span> may (a) access and use the Cloud Service; and (b) copy and use the included Software and Documentation only as needed to access and use the Cloud Service, in each case, for its internal business purposes and only if <span class="keyterms_link">Customer</span> complies with the terms of this Agreement.
 
-    1.2 <span class="header_3">User Accounts.</span>  <span class="coverpage_link">Customer</span> is responsible for all actions on its Users' accounts and for its Users' compliance with this Agreement. <span class="coverpage_link">Customer</span> and its Users must protect the confidentiality of its passwords and login credentials. <span class="coverpage_link">Customer</span> will promptly notify <span class="coverpage_link">Provider</span> if it suspects or knows of any fraudulent activity with its accounts, passwords, or credentials, or if they become compromised.
+    1.2 <span class="header_3">Service Level.</span> If the Cloud Service does not meet the <span class="orderform_link">SLA</span>, if any, <span class="keyterms_link">Provider</span> will provide the remedies outlined in the <span class="orderform_link">SLA</span> and will not be responsible for any other remedies. If there is an <span class="orderform_link">SLA</span>, any credits earned under the <span class="orderform_link">SLA</span> will only apply to future invoices and expire if the Agreement ends. In any event, if the Cloud Service is temporarily unavailable for scheduled maintenance, for unscheduled emergency maintenance, or because of other causes beyond <span class="keyterms_link">Provider's</span> reasonable control, no <span class="orderform_link">SLA</span> remedies will accrue. <span class="keyterms_link">Provider</span> will try to inform <span class="keyterms_link">Customer</span> before scheduled service disruptions through the Cloud Service or by email.
```

Comment:
Grammatical change to "if any" approach

```suggestion
    1.2 <span class="header_3">Service Level.</span> If there is an <span class="orderform_link">SLA</span> and the Cloud Service does not meet the <span class="orderform_link">SLA</span>, <span class="keyterms_link">Provider</span> will provide the remedies outlined in the <span class="orderform_link">SLA</span> and will not be responsible for any other remedies. Any credits earned under the <span class="orderform_link">SLA</span> will only apply to future invoices and expire if the Agreement ends. In any event, if the Cloud Service is temporarily unavailable for scheduled maintenance, for unscheduled emergency maintenance, or because of other causes beyond <span class="keyterms_link">Provider's</span> reasonable control, no <span class="orderform_link">SLA</span> remedies will accrue. <span class="keyterms_link">Provider</span> will try to inform <span class="keyterms_link">Customer</span> before scheduled service disruptions through the Cloud Service or by email.
```